apl-core/src/main/java/com/apollocurrency/aplwallet/apl/core/rest/v2/impl/SmcApiServiceImpl.java

```diff
@@ -855,16 +710,15 @@ public Response getSmcList(String addressStr, String publisherStr, String name,
 
         FirstLastIndexBeanParam indexBeanParam = new FirstLastIndexBeanParam(firstIndex, lastIndex);
         indexBeanParam.adjustIndexes(maxAPIRecords);
-        AplAddress address = null;
-        AplAddress publisher = null;
-        AplAddress transaction = null;
+        Long address = null;
+        Long publisher = null;
+        Long transaction = null;
 
-        ContractStatus smcStatus = null;
         if (status != null) {
             try {
-                smcStatus = ContractStatus.valueOf(status);
+                ContractStatus.valueOf(status);
```

Comment:
Is that ok ? Is status only checked for correctness and not assigned to any var ?

Reply:
Yes,  the 'status' param is used below as a string